design/src/main/res/values/belchior_attrs.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- BUTTONS -->
+    <declare-styleable name="DSEventSessionButton">
+        <attr name="week_day" />
+        <attr name="date" />
+        <attr name="hour" />
+        <attr name="enabled" format="boolean"/>
```

Comment:
These need to be styleable attributes? I think these could be inside the class, not style properties. Please, see what is necessary to declared in XML and what can be declared inside each component in the class file.